### DISCUSSION TITLE
Dialoge question even closing with any click

### DISCUSSION THREAD

**Discussion by Redmouse9** [dialogic-godot/dialogic#844] [Q&A]
Hello everyone. I'm creating a question event

![image](https://user-images.githubusercontent.com/95829450/159588789-e79d59fd-c46c-4466-8b90-ecb9f77c889d.png)

and when I run the event it will close the question dialogue even if the click is not on any of the choices, such as a mis-click. Is there a way to stay with the question dialogue until one is selected?

**Comment by coppolaemilio** [comment]
The choices need to have events under them. Even if that event is "Close Dialog" you need to have something under each of those options you have there!

**Reply by Redmouse9** [reply]
Half way there... I added a set value to each. Now mis-click defaults to the first choice on the list.

![image](https://user-images.githubusercontent.com/95829450/160021377-d2c48a26-e12a-4522-827d-2f69d5a489dc.png)

**Reply by Jowan-Spooner** [reply]
Make sure you haven't added the left mouse button to the ui_accept action in the input map. You should use a separate input action for the dialog, because ui_* actions are used by buttons and other control elements.

**Reply by Redmouse9** [reply]
Thanks for your quick reply. :) The left mouse button is not mapped to ui_accept. I can get the same effect using the other dialogic_default_action keys as well. It selects the first item listed.

**Reply by radix-r** [reply]
Just noticed that this is the same issue I bring up in #858. I included a video and a supplementary description of the problem that my help clarify any confusion.

**Reply by Redmouse9** [reply]
Thank you. This was helpful.
